projects/angular/src/forms/datepicker/_datepicker.clarity.scss

```diff
@@ -74,8 +93,36 @@
     border: tokens.$cds-alias-object-border-width-100 solid datepicker-variables.$clr-calendar-border-color;
     border-radius: tokens.$cds-alias-object-border-radius-100;
     box-shadow: tokens.$cds-alias-object-shadow-100;
-    overflow: hidden;
     z-index: map.get(variables.$clr-layers, dropdown-menu);
+    justify-content: space-between;
```

Comment:
Do we need that?

Reply:
Related to daterange, will remove it here & add it in range picker PR.